extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -33,6 +34,13 @@
     @ConfigItem(defaultValue = "false")
     public boolean ignoreStaticDocument;
 
+    /**
+     * A list of local directories that should be scanned for yaml and/or json files to be included in the static model.
+     * Example: `META-INF/openapi/`
+     */
+    @ConfigItem
+    public Optional<List<Path>> additionalStaticDocumentsDirectory;
```

Comment:
This is kind of a mouthful :). Maybe just `additionalDocsDirectory`?

Reply:
Cool, I'll change that.

Reply:
Changed